<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color:white;</string>
  </property>
  <widget class="QLabel" name="automata_output">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>521</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
border: 5px solid lightgreen;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="solve_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
    background-color: orange;
	color: white;
}

QPushButton{
background-color: rgb(170, 255, 127);
border-radius:10px;
color:black solid;
}

</string>
   </property>
   <property name="text">
    <string>Get Automata</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>22</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
font: 75 22pt &quot;MS Shell Dlg 2&quot;;
padding: 10 px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Grammar -&gt; Automata</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
    background-color: lightgreen;
	color: white;
}
QPushButton{
font: 75 15pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:20px;
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
    background-color: black;
	color: white;
}
QPushButton{
background-color: rgb(203, 203, 203);
color:black;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
border:2px solid lightgreen;
}
</string>
   </property>
   <property name="text">
    <string>RESET GRAMMAR</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="grammar_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
font: 12pt &quot;MS Shell Dlg 2&quot;;
color:black;</string>
   </property>
   <property name="placeholderText">
    <string>Enter grammar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>401</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="title">
    <string>Add Production</string>
   </property>
   <widget class="QLineEdit" name="head_le">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit:hover{
border:2px solid lightgreen;
}

QLineEdit{
font:  12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color:black;
border:2px solid gray;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Head</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="body_le">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit:hover{
border:2px solid lightgreen;
}

QLineEdit{
font:  12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color:black;
border:2px solid gray;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Body</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_production_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>105</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
    background-color: black;
	color: white;
}
QPushButton{
background-color: rgb(203, 203, 203);
color:black;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border:2px solid lightgreen;
}
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
